products/ceqr_survey/models/intermediate/int__dep_cats_permits.sql

```diff
@@ -16,11 +16,9 @@ pluto AS (
 
 cats_permits_with_pluto AS (
     SELECT
-        s.variable,
-        s.id,
-        s.permit_geom,
-        p.geom AS bbl_geom
-
+        s.variable AS variable_type,
```

Comment:
Doesn't have to be addressed in this PR: 

I'm leaning towards renaming columns in staging rather than in intermediate tables for consistency (I believe some tables are renamed in staging) and readability.

Reply:
would def be interested to get everyone's thoughts on this

Reply:
+1 for doing it in staging

Reply:
Though I sort of see the rationale for keeping those more closely tied to the source data. I'm in the middle

Reply:
let's leave it for discussion next week?

Reply:
Well actually let's at least make it one way or the other - right now it's "variable" in staging, which doesn't align with source data set nor with the intermediate tables.

Let's go variable_type in stg for now?

Reply:
if we change variable_type in stg, then lets revise variable_id there as well

Reply:
moved all the renames to stg__*, when possible